### DISCUSSION TITLE
Reference Architecture on Terraform Cloud

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#454] [Help] [answered]
If I'm a Terraform Cloud user, is it possible to deploy the Reference Architecture or migrate its state into TFC?

---

<ins datetime="2022-06-09T20:18:02Z">
  <p><a href="https://support.gruntwork.io/hc/requests/108737">Tracked in ticket #108737</a></p>
</ins>

**Comment by yorinasub17** [comment, marked as answer]
We don't support Terraform Cloud with the Reference Architecture, but we are considering this on our roadmap.

**Reply by jbrooks-sms** [reply]
Is Terraform Cloud with Reference Architecture still not supported or has this been added since this last response?  Thank you

**Reply by iantybo** [reply]
Hello @jbrooks-sms, it is not supported at this time. Please feel free to follow up with me directly at sales@gruntwork.io with any questions.

**Reply by jbrooks-sms** [reply]
Thanks for the confirmation @iangrunt
